README.md

```diff
@@ -4,30 +4,114 @@ As enterprise businesses embrace Machine Learning (ML) across their organisation
+To get started we recommend leveraging the helper scripts in [scripts](mlops-multi-account-cdk/scripts) which will help you installing prerequisites on your machine, create the appropriate config files, and bootstrap your AWS accounts.
+If you would rather do those actions manually skip the section below and refer to the instructions in [mlops-infra](mlops-multi-account-cdk/mlops-infra/README.md) and [mlops-sm-project-template](mlops-multi-account-cdk/mlops-sm-project-template/README.md).
+
+### First step: Setup AWS Profiles
+
+As the MLOps foundation is based on multiple accounts, it is necessary to create a simple way to interact with multiple AWS credentials. We recommend the creation of an AWS profile per account with enough permission to deploy to CloudFormation following the instructions [here](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/keys-profiles-credentials.html#adding-a-profile-to-the-aws-credentials-profile-file) . For example, the `.aws/credentials` should look like:
+
+```
+[mlops-governance]
+aws_access_key_id = YOUR_ACCESS_KEY_ID
+aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
+aws_session_token = YOUR_SESSION_TOKEN  # this token is generated if you are using an IAM Role to assume into the account
+
+[mlops-dev1]
+aws_access_key_id = YOUR_ACCESS_KEY_ID
+aws_secret_access_key = YOUR_SECRET_ACCESS_KEY
+aws_session_token = YOUR_SESSION_TOKEN  # this token is generated if you are using an IAM Role to assume into the account
+
+[mlops-preprod1]
+...
+
+[mlops-prod1]
+...
+```
+
+### Second step: install prerequisites
+
+```bash
+mlops-multi-account-cdk/scripts/quick-start.sh prerequisites
+```
+
+### Third step: create config files
+
+```bash
+mlops-multi-account-cdk/scripts/quick-start.sh config
+```
+
+This will create for you the required governance account and business unit config files in both [mlops-infra](mlops-multi-account-cdk/mlops-infra/config/) and [mlops-sm-project-template](mlops-multi-account-cdk/mlops-sm-project-template/config/)
+It will also generate a temporary config file `bootstrap_conf.tmp`  that will be used to bootstrap the different accounts.
+Please note, the prompt will ask you for an AWS profile for each account. This should correspond to what you defined in the first step "Setup AWS Profiles".
+
+The script will request the at least 4 accounts, i.e. governance, and one business-unit composed of dev, preprod and prod, and the corresponding AWS profiles as inputs. If you want to only deploy to 1 account you can use the same id for all account variables or pass the same values in the script.
+
+### Fourth step: boostrap the accounts
+
+```bash
+mlops-multi-account-cdk/scripts/quick-start.sh bootstrap
+```
+
+### Fifth step: cdk deploy
+
+```bash
+cd mlops-multi-account-cdk/mlops-infra/
+pip install -requirements.txt
+cdk deploy --all --profile mlops-governance
+```
+
+```bash
+cd mlops-multi-account-cdk/mlops-sm-project-template/
```

Comment:
If there are additional python modules which are used in mlops-sm-project-template then it is obvious that we should also include
pip install -r requirements.txt